src/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+div {
+  width: 300px;
+  height: 100px;
+  margin: auto;
+}
+
+div:hover {
+  height: 200px;
+}
+
+#header {
+  background-color: blue;
+  height: 100px;
+}
+
+.important {
+  background-color: red;
+}
+
+.interesting {
+  background-color: green;
+}
+
+.surprising {
+  background-color: orange;
+}
+
+#footer {
+  background-color: black;
+  height: 100px;
```

Comment:
When hovered (the cursor is on the block), height must increase to 200 pixels.

Reply:
I was thinking that header and footer should not be mutable. I'll fix it